backend/src/main/kotlin/com/woowahan/campus/announcement/domain/Author.kt

```diff
@@ -3,27 +3,13 @@ package com.woowahan.campus.announcement.domain
 import jakarta.persistence.Embeddable
 
 @Embeddable
-class Author(val author: String) {
+data class Author(val author: String) {
     init {
-        val trimmedAuthor = author.trim()
-        if (trimmedAuthor.isEmpty() || trimmedAuthor.length > MAX_LENGTH) {
-            throw IllegalArgumentException("공지의 작성자는 1자 이상, 20자 이하로 작성 가능합니다.")
+        require(author.isNotEmpty() && author.length <= MAX_LENGTH) {
+            "공지의 작성자는 1자 이상, 20자 이하로 작성 가능합니다."
```

Comment:
```suggestion
            "공지의 작성자는 1자 이상, ${MAX_LENGTH}자 이하로 작성 가능합니다." 
```
문자열 템플릿을 활용해보는건 어떨까요? 😊